### PR TITLE
Update water.markdown

### PR DESCRIPTION
**Pull request by martijnvwezel**
## Proposed change
This pull request addresses the issue of clarity for new users who may find the updated version vague and struggle to know where to begin. To enhance user experience, I have made clear distinctions between integration and community-built sensors or do-it-yourself (DIY) versions. This will provide newcomers with a clearer roadmap and understanding of how to engage with the project effectively.

## Type of change

- [X ] Spelling, grammar or other readability improvements (`current` branch).
- [ X] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Checklist

- [X ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [X ] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_docs/energy/water.markdown [hunk 1]
```diff
@@ -18,30 +26,28 @@ We have the following integrations available for existing products that can prov
...
-- [P1 Monitor](/integrations/p1_monitor)
 - [StreamLabs](/integrations/streamlabswater)
 - [Suez Water](/integrations/suez_water)
 
-There are also products for water usage monitoring that are based on existing common IoT protocol standards:
+## Community Made Sensors
+If your water meter lacks a rotary disk, magnetic disk, or coil. There are alternative solutions available to seamlessly integrate water monitoring into your smart home setup:
+- [AI-on-the-edge-device](https://github.com/jomjol/AI-on-the-edge-device) is a project running on an ESP32-CAM and can be fully integrated into Home Assistant using the Home Assistant Discovery Functionality of MQTT. It digitalizes your gas/water/electricity meter display and provides its data in various ways.![Photo of the AI-on-the-edge-device Workflow](/images/docs/energy/
 
-- [Z-Wave](/integrations/zwave_js)
```
Comment (c0ffeeca7) [inline source/_docs/energy/water.markdown]: why are these removed?

Comment (martijnvwezel) [reply source/_docs/energy/water.markdown]: I removed the Z-Wave details because they shift focus from our main narrative to technical specifics, better suited for a DIY guide. This keeps our content focused and relevant.

Comment (frenck) [reply source/_docs/energy/water.markdown]: This is not a DIY-guide.
As a matter of fact, non-DIY should be the main focus.

Please restore these, and put them above the DIY solutions.

Comment (martijnvwezel) [reply source/_docs/energy/water.markdown]: I agree with your perspective, highlighting Z-Wave doesn't necessarily clarify the narrative. It remains unclear to me why their inclusion is essential. However, I will proceed to reintegrate them into the text as requested.

File: source/_docs/energy/water.markdown [hunk 2]
```diff
@@ -5,10 +5,18 @@ description: "Learn how to add information about your water usage to Home Assist
 
 Home Assistant allows you to track your water usage in the home energy management too.
 
-Although water usage is not strictly "energy", it is still a valuable resource to track and monitor as it is often tightly coupled with energy usage (like gas). Additionally, it can help you reduce your ecological footprint by using less water. 
+Although water usage is not strictly "energy", it is still a valuable resource to track and monitor as it is often tightly coupled with energy usage (like gas). Additionally, it can help you reduce your ecological footprint by using less water.
 
-## Hardware
+### Home water meters
```
Comment (c0ffeeca7) [inline source/_docs/energy/water.markdown:10]:
```suggestion
### Home water meters

```

File: source/_docs/energy/water.markdown [hunk 3]
```diff
@@ -5,10 +5,18 @@ description: "Learn how to add information about your water usage to Home Assist
...
 
+Newer water meters are equipped with a rotary disk that can be read using two methods. The first method utilizes light sensors, while the second method employs proximity sensors. The proximity sensor detects changes in the magnetic field, with each rotation of the disk representing one liter of water used. Meanwhile, the light sensor method operates on an autocorrelation technique, providing accuracy down to 100 milliliters instead of the traditional one-liter step.
+
+For most water meters, the rotary encoder disk suffices the light sensor version. However, some older or specialized meters may necessitate the use of a proximity meter instead.
+
+Home Assistant also has integrations build into the platform that connect with existing products
+
+## Home Assistant integrations
```
Comment (c0ffeeca7) [inline source/_docs/energy/water.markdown:20]:
```suggestion
## Home Assistant integrations

```

File: source/_docs/energy/water.markdown [hunk 4]
```diff
@@ -18,30 +26,34 @@ We have the following integrations available for existing products that can prov
...
 
 There are also products for water usage monitoring that are based on existing common IoT protocol standards:
-
 - [Z-Wave](/integrations/zwave_js)
 - [Zigbee](/integrations/zha)
 - [Matter (BETA)](/integrations/matter)
 
-Alternatively, the following shops sell ESPHome-based devices, that use a proximity sensor to detect a rotating magnet in your water meter and use that pulse to count each liter of water used.
 
+## Community Made Sensors
```
Comment (c0ffeeca7) [inline source/_docs/energy/water.markdown]:
```suggestion
## Community-made sensors

```

File: source/_docs/energy/water.markdown [hunk 5]
```diff
@@ -18,30 +26,34 @@ We have the following integrations available for existing products that can prov
...
+Alternatively, the following shops sell ESPHome-based devices, that use a proximity sensor to detect a rotating magnet in your water meter and use that pulse to count each liter of water used:
 - [S0tool](https://s0tool.nl/) ("Made for ESPHome" approved)
 - [Waterlezer dongle](https://smart-stuff.nl/product/esphome-waterlezer-dongle/) (Dutch)
 - [Slimme Watermeter Gateway](https://smartgateways.nl/product/slimme-watermeter-gateway/) (Dutch)
 - [watermeterkit.nl](https://watermeterkit.nl/) (Dutch)
 
-Alternatively, the following shops sell ESPHome-based devices that use a 3-phase light sensor to detect a rotating disk in your water meter and convert this to the amount of water used in milliliters (ml).
-- [Muino water meter reader](https://watermeter.muino.nl/)
-
+## DIY
```
Comment (c0ffeeca7) [inline source/_docs/energy/water.markdown:56]:
```suggestion
## DIY

```

File: source/_docs/energy/water.markdown [hunk 6]
```diff
@@ -18,30 +26,34 @@ We have the following integrations available for existing products that can prov
 - [Flo](/integrations/flo)
 - [Flume](/integrations/flume)
 - [HomeWizard Energy](/integrations/homewizard)
-- [P1 Monitor](/integrations/p1_monitor)
 - [StreamLabs](/integrations/streamlabswater)
 - [Suez Water](/integrations/suez_water)
 
 There are also products for water usage monitoring that are based on existing common IoT protocol standards:
```
Comment (c0ffeeca7) [inline source/_docs/energy/water.markdown]:
```suggestion
There are also products for water usage monitoring that are based on existing common IoT protocol standards:

```

File: source/_docs/energy/water.markdown [hunk 7]
```diff
@@ -18,30 +26,34 @@ We have the following integrations available for existing products that can prov
...
-
 - [Z-Wave](/integrations/zwave_js)
 - [Zigbee](/integrations/zha)
 - [Matter (BETA)](/integrations/matter)
 
-Alternatively, the following shops sell ESPHome-based devices, that use a proximity sensor to detect a rotating magnet in your water meter and use that pulse to count each liter of water used.
 
+## Community Made Sensors
+If your water meter lacks a rotary disk, magnetic disk, or coil. There are alternative solutions available to seamlessly integrate water monitoring into your smart home setup:
```
Comment (c0ffeeca7) [inline source/_docs/energy/water.markdown:43]:
```suggestion
If your water meter lacks a rotary disk, magnetic disk, or coil. There are alternative solutions available to seamlessly integrate water monitoring into your smart home setup:

```

File: source/_docs/energy/water.markdown [hunk 8]
```diff
@@ -18,30 +26,34 @@ We have the following integrations available for existing products that can prov
...
+- Pieter Brinkman has quite a [nice blog article on how to create your own water sensor](https://www.pieterbrinkman.com/2022/02/02/build-a-cheap-water-usage-sensor-using-esphome-home-assistant-and-a-proximity-sensor/) using ESPHome, or [build a water meter](https://www.ztatz.nl/p1-monitor-watermeter/) that works with the [P1 Monitor](/integrations/p1_monitor) integration.
+- [AI-on-the-edge-device](https://github.com/jomjol/AI-on-the-edge-device) is a project running on an ESP32-CAM and can be fully integrated into Home Assistant using the Home Assistant Discovery Functionality of MQTT. It digitalizes your gas/water/electricity meter display and provides its data in various ways.![Photo of the AI-on-the-edge-device Workflow](/images/docs/energy/ai-on-the-edge-device.jpg)
+- [watermeter](https://github.com/nohn/watermeter) running classic OCR and statistical pattern recognition on any system supporting Docker
+- [Muino water meter reader 3-phase](https://muino.nl/product/3-phase-muino-light-sensor-encoder/) Using the 3phase sensor technique, battery powered version can be possible with this sensor.
```
Comment (c0ffeeca7) [inline source/_docs/energy/water.markdown]:
```suggestion
- [Muino water meter reader 3-phase](https://muino.nl/product/3-phase-muino-light-sensor-encoder/) Using the 3-phase sensor technique, a battery-powered version can be possible with this sensor.
```

**Review by c0ffeeca7 — COMMENTED** [review]
Tiny style tweaks

**Review by c0ffeeca7 — APPROVED** [review]
Thank you, @martijnvwezel :+1:
